dist/debian/control

```diff
@@ -22,6 +22,7 @@ Build-Depends:
  libfuse3-dev,
  zlib1g-dev,
  dh-apparmor
+Recommends: libsubid-dev
```

Comment:
I think this should be Build-Recommends

Reply:
There is no Build-Recommends in debian control? In fact the Recommends control field is probably just ignored for src packages.  Should we just leave this out? or we can add it to Build-Depends?

Reply:
You're right.  I thought I saw it when I checked, but there are only discussions about it and it was never added.  They also don't have conditionals.  So I think the best we can do right now is leave it out, but add some instructions in dist/debian/DEBIAN_PACKAGE.md on how to add it in on the relevant debian and ubuntu distributions.

Reply:
Sounds good, all set. added some docs to DEBIAN_PACKAGE.md